scripts/who-am-i/src/platform.ts

```diff
@@ -0,0 +1,17 @@
+import type { TupleValues } from './utils';
+
+export const knownPlatforms = [
+    'HTML5',
+    'WECHAT',
+    'ALIPAY',
+    'BAIDU',
+    'XIAOMI',
+    'BYTEDANCE',
+    'OPPO',
+    'VIVO',
+    'HUAWEI',
+    'NATIVE',
+    'COCOSPLAY',
+] as const;
```

Comment:
Platform related. @yanOO1497  @LinYunMo @PPpro

Reply:
need 
QTT and LINKSURE